sdk/identity/azure-identity/README.md

```diff
@@ -138,7 +138,7 @@ client secret and certificate are both present, the client secret will be used.
 ## Authenticating with `DefaultAzureCredential`
 This example demonstrates authenticating the `BlobServiceClient` from the
 [azure-storage-blob][azure_storage_blob] library using
-[DefaultAzureCredential][default_cred_ref].
+[DefaultAzureCredential](#defaultazurecredential "DefaultAzureCredential").
```

Comment:
This one replace an aka.ms link by a local link, on purpose?

Reply:
Yes, I hope readers will see the `DefaultAzureCredential` content here before the reference docs.